spec/controllers/ops_controller_spec.rb

```diff
@@ -281,36 +281,6 @@
     end
   end
 
-  context "Toolbar buttons render" do
```

Comment:
I'm not sure about removing this spec - Not sure if it's testing this tab, or just happens to be using this tab to test toolbar button rendering.

Reply:
I searched the history and this test was added specifically for log collection, so we can drop it.